docs/esp8266/general.rst

```diff
@@ -18,8 +18,8 @@ To make a generic ESP8266 port and support as many boards as possible,
 following design and implementation decision were made:
 
 * GPIO pin numbering is based on ESP8266 chip numbering, not some "logical"
-  numbering of a particular board. Please have manual/pin diagram of your board
-  handy to find correspondce between your board pins and actual ESP8266 pins.
+  numbering of a particular board. Please have the manual/pin diagram of your board
+  handy to find diversions between your board pins and actual ESP8266 pins.
```

Comment:
Actually, diversions isn't right. I'd either stick with diversions or perhaps use "the mapping".